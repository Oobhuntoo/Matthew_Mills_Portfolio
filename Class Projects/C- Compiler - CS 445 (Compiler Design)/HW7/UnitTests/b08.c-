main()
{
     outputb(not true);
     outnl();
     outputb(not false);
     outnl();
}
