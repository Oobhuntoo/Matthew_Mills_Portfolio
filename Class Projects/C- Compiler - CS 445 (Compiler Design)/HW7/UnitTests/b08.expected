Loading file: b08.tm
F 
T 

Status: Halted
Bye.
